examples/data-sources/snowflake_secrets/data-source.tf

```diff
@@ -0,0 +1,46 @@
+# Simple usage
+data "snowflake_secrets" "simple" {
+}
+
+output "simple_output" {
+  value = data.snowflake_secrets.simple.secrets
+}
+
+# Filtering (like)
+data "snowflake_secrets" "like" {
+  like = "secret-name"
+}
+
+output "like_output" {
+  value = data.snowflake_secrets.like.secrets
+}
+
+# Filtering by prefix (like)
+data "snowflake_secrets" "like_prefix" {
+  like = "prefix%"
+}
+
+output "like_prefix_output" {
+  value = data.snowflake_secrets.like_prefix.secrets
+}
+
+# Filtering (in)
+data "snowflake_secrets" "in" {
+  in {
+    schema = "schema_fully_qualified_name"
```

Comment:
I think we try to use `fully_qualified_name` computed values instead of strings, so this should be `schema = snowflake_schema.test.fully_qualified_name`, check other examples on that

Reply:
changed it